MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: Getting Started with .NET MAUI DataForm control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI DataForm (SfDataForm) control and its basic features.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Getting Started with .NET MAUI DataForm (SfDataForm)
+
+This section provides a quick overview of how to get started with the .NET MAUI DataForm(SfDataForm) for .NET MAUI and also provides a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
```

Comment:
This section provides a quick overview of how to get started with the .NET MAUI DataForm(SfDataForm) for .NET MAUI and a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.

Reply:
content updated